app/controllers/services_controller.rb

```diff
@@ -55,14 +55,15 @@ def other_country_message
 
   def hidden_user_explanation
     info_request = InfoRequest.find(params[:info_request_id])
-    render :template => "admin_request/hidden_user_explanation",
-      :content_type => "text/plain",
-      :layout => false,
-      :locals => {:name_to => info_request.user_name,
-                  :name_from => AlaveteliConfiguration.contact_name,
-                  :info_request => info_request, :reason => params[:reason],
-                  :info_request_url => 'http://' + AlaveteliConfiguration.domain + request_path(info_request),
-                  :site_name => site_name}
+    render template: "admin_request/hidden_user_explanation",
+           formats: [:text],
+           layout: false,
+           locals: {
+             name_to: info_request.user_name,
+             name_from: AlaveteliConfiguration.contact_name,
+             info_request: info_request, :reason => params[:reason],
```

Comment:
You missed `:reason => params[:reason]` – should drop that on to a new line and make it a 1.9-style hash